### DISCUSSION TITLE
Connecting from different PostgreSQL clients i.e. pgAdmin fails with "received invalid response to GSSAPI negotiation: S"

### DISCUSSION THREAD

**Discussion by k-ison** [supabase/supabase#30173] [Troubleshooting]
This error happens due to a missing parameter that needs to be set to disabled.

There are few ways to disable the "gssencmode" parameter:
1. Connection configuration file.
- [ add gssencmode = 'disable'] to your configuration file
2. In pgAdmin, when registering a server.
- [ Go to Parameters ]
- [ Click the + button ]
- [ From the dropdown, select 'GSS encmode' ]  
- [ Set the value to 'disable' ]
- [ Fill out all required connection details then save ]

**Comment by TeemuKoivisto** [comment]
I was able to connect with `psql` and postgresql 14.14 to supabase from my MBP M1 perfectly, yet as I upgraded to 15.9 I immediately started receiving this error. Upgrading postgresql all the way to 17 fixed it.

**Reply by erpview** [reply]
Form this worked  also

**Reply by xxxoooxoxo** [reply]
Worked for me, as well

**Reply by Priyank-Maniya** [reply]
worked for me, Thanks !

**Comment by Saurabh0377** [comment]
Visit 

 https://agentic-qa-api.onrender.com/docs

**Comment by Saurabh0377** [comment]
https://agentic-qa-api.onrender.com/docs
